nomad/structs/acl.go

```diff
@@ -1378,9 +1493,38 @@ func (a *ACLOIDCCompleteAuthRequest) Validate() error {
 	return mErr.ErrorOrNil()
 }
 
-// ACLOIDCCompleteAuthResponse is the response when the OIDC auth flow has been
+// ACLCompleteAuthResponse is the response when the auth flow has been
 // completed successfully.
-type ACLOIDCCompleteAuthResponse struct {
+type ACLCompleteAuthResponse struct {
 	ACLToken *ACLToken
 	WriteMeta
 }
+
+// ACLLoginRequest is the request object to begin auth with an external bearer
+// token provider.
+type ACLLoginRequest struct {
+
+	// AuthMethodName is the name of the auth method being used to login. This
+	// is a required parameter.
+	AuthMethodName string
+
+	// LoginToken is the token used to login. This is a required parameter.
```

Comment:
Would you be able to expand where this token comes from? It's not clear to me exactly where this comes from, and whether this is different from a Nomad ACL token.